rr_computer_vision/include/traffic_light.hpp

```diff
@@ -1,25 +1,53 @@
 /**
  * @file rr_traffic_light.hpp
  * @brief Traffic Light Header File
- * @author Jamie Kim
- * @author Jason Leung
- * @author Toni Ogunmade(oluwatoni)
- * @author Adrian Malaran
- * @competition IARRC 2018
+ * @author Yuchi(Allan) Zhao
+ * @competition IARRC 2019
  */
 
-#ifndef TRAFFIC_LIGHT
-#define TRAFFIC_LIGHT
+#ifndef RR_TRAFFIC_LIGHT
+#define RR_TRAFFIC_LIGHT
 
 // ROS includes
 #include <ros/ros.h>
+#include <image_transport/image_transport.h>
+#include <cv_bridge/cv_bridge.h>
+#include <sensor_msgs/image_encodings.h>
+#include <ros/console.h>
+#include <std_msgs/Int8.h>
+#include <std_msgs/Float32.h>
+#include <std_srvs/Empty.h>
+#include <std_srvs/Trigger.h>
 
-class TrafficLight {
+// OPENCV includes
+#include <opencv2/opencv.hpp>
+
+#include <sensor_msgs/Image.h>
+
+
+class TrafficLightProcessor
```

Comment:
I know the name of this class is legacy from last year's code, but I actually think this ought to be called 'TrafficLightDetection' for clarity, processor is a little vague and doesn't convey the information as well in my opinion.